com.Cisco.PacketTracer.metainfo.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.Cisco.PacketTracer</id>
+  <name>Packet Tracer</name>
+  <launchable type="desktop-id">com.Cisco.PacketTracer.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Cisco Systems, Inc</developer_name>
+  <summary>Gain real skills with our powerful network simulation tool where you practice networking, IoT, and cybersecurity skills in a virtual lab. No hardware needed!</summary>
+  <description>
+    <p>Packet Tracer is a cross-platform visual simulation tool designed by Cisco Systems that allows users to create network topologies and imitate modern computer networks.</p>
+    <p>The software allows users to simulate the configuration of Cisco routers and switches using a simulated command line interface.</p>
+    <p>Packet Tracer makes use of a drag and drop user interface, allowing users to add and remove simulated network devices as they see fit.</p>
+    <p>The software is mainly focused towards Cisco Networking Academy students as an educational tool for helping them learn fundamental CCNA concepts.</p>
+    <p>Previously students enrolled in a CCNA Academy program could freely download and use the tool free of charge for educational use.</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Cisco System, Inc.</p>
+  </description>
+  <url type="homepage">https://www.netacad.com/courses/packet-tracer</url>
```

Comment:
why doesn't the homepage match the app-id domain?

Reply:
Cisco.com is the main Cisco website, the nedacad.com is their learning/courses website and where Packet Tracker is referenced.